content/pages/health-care/health-conditions/mental-health/ptsd.md

```diff
@@ -31,23 +31,22 @@ Our National Center for PTSD is the world leader in PTSD research, education, an
 
 </div>
 
-<div class="usa-alert usa-alert-warning va-alert va-crisis-alert">
+<div class="usa-alert usa-alert-warning">
   <div class="usa-alert-body">
-	<h4 class="va-alert-title">Are you in crisis? <a id="crisis-expander-link">You have options.</a></h4>
+	 <h4 usa-alert-title>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h4>
 	<div id="crisis-expander-content" class="expander-content expander-content-closed">
```

Comment:
Looks like this one is missing the class part too

Reply:
Good catch! I'm not so worried about matching uswds standards on this yet, but that's probably a next step for these.